interfaces/udev/udev.go

```diff
@@ -45,8 +61,7 @@ func (b *Backend) reloadRules(subsystemTriggers []string) error {
 	// By default, trigger for all events except the input subsystem since
 	// it can cause noticeable blocked input on, for example, classic
 	// desktop.
-	output, err = exec.Command("udevadm", "trigger", "--subsystem-nomatch=input").CombinedOutput()
-	if err != nil {
+	if err = udevadmTrigger("--subsystem-nomatch=input"); err != nil {
 		return fmt.Errorf("cannot run udev triggers: %s\nudev output:\n%s", err, string(output))
```

Comment:
On error we're going to print the output of `udevadm control ...` now, not the out[ut of trigger command, is this intended?

Reply:
Silly me - of course not. Thanks for noticing :)